scripts/Readme.md

```diff
@@ -1,7 +1,7 @@
 # Scripts files for GDevelop
 
-* **ReleaseProcedure.bat**: this script compiles, generate documentation (see **GenerateAllDocs.bat**) and package GDevelop for Windows in an installer and an archive.
-* **ReleaseProcedure.sh**: compiles and package GD for Ubuntu (see *Binaries/Packaging*).
-* **CopyWindowsToLinuxReleaseFiles.sh**: Copy all files in *Binaries/Output/Release_Windows* to *Binaries/Output/Release_Linux*. Call it after any change in *Binaries/Output/Release_Windows*.
-* **GenerateAllDocs.[bat|sh]**: Call doxygen and yuidoc to generate all the documentations into *docs* folder.
-* **ExtractTranslations.[bat|sh]**: Create the *source.pot* file containing the strings to be translated using [Crowdin](https://crowdin.com/project/gdevelop).
+-   **ReleaseProcedure.bat**: this script compiles, generate documentation (see **GenerateAllDocs.bat**) and package GDevelop for Windows in an installer and an archive.
```

Comment:
Do yo know why there are some spaces after the - in some files (like here: `-   `) and not in others?

Reply:
This is just the prettier formatting. There is no impact on the preview.